### PR TITLE
balance: drop unused Mint/Burn notifications, fix #318

### LINKED ISSUE #318: Refactor notifications emitted by balance contract
Notifications `Mint` and `Burn` can be removed from the contract code as there's no users of it and (if needed) minting\burning can easily be tracked from the `Transfer` notifications with nil sender\receiver correspondingly.

Ref. #304.

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: balance/balance_contract.go [hunk 1]
```diff
@@ -307,7 +306,6 @@ func Burn(from interop.Hash160, amount int, txDetails []byte) {
 	supply = supply - amount
 	storage.Put(ctx, token.CirculationKey, supply)
 	runtime.Log("assets were burned")
-	runtime.Notify("Burn", from, amount)
```
Comment (AnnaShaleva) [inline balance/balance_contract.go:310]: > // It produces Burn, Transfer and TransferX notifications.

**Review by AnnaShaleva — COMMENTED** [review]
Otherwise LGTM.

File: balance/balance_contract.go [hunk 2]
```diff
@@ -272,7 +272,6 @@ func Mint(to interop.Hash160, amount int, txDetails []byte) {
 	supply = supply + amount
 	storage.Put(ctx, token.CirculationKey, supply)
 	runtime.Log("assets were minted")
-	runtime.Notify("Mint", to, amount)
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply balance/balance_contract.go:275]: Fixed.

File: balance/balance_contract.go (hunk 1 above)
Comment (roman-khimov) [reply balance/balance_contract.go:310]: Fixed.

File: balance/config.yml [hunk 3]
```diff
@@ -34,15 +34,3 @@ events:
...
-      - name: to
-        type: Hash160
-      - name: amount
-        type: Integer
-  - name: Burn
-    parameters:
-      - name: from
-        type: Hash160
-      - name: amount
-        type: Integer
```
Comment (AnnaShaleva) [inline balance/config.yml:48]: No new line in the end of the file.

**Review by AnnaShaleva — APPROVED** [review]

File: balance/config.yml (hunk 3 above)
Comment (roman-khimov) [reply balance/config.yml:48]: In the old file, now it's there!

**Review by cthulhu-rider — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Pull request merged**